.github/workflows/nodejs.yml

```diff
@@ -10,12 +10,12 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  build-and-test:
```

Comment:
Why does this command need to change?

Reply:
That's just the name of the job. I renamed it to be more descriptive. It's not just building, it's building and testing.